custom/icds_reports/tests/test_disha_dump.py

```diff
@@ -5,13 +5,15 @@
 import mock
 
 from datetime import date
-from django.test.testcases import SimpleTestCase
 from io import open
+
+from django.test import TestCase
+
 from corehq.util.files import TransientTempfile
 from custom.icds_reports.reports.disha import DishaDump
 
 
-class DishaFileBuildTest(SimpleTestCase):
+class DishaFileBuildTest(TestCase):
```

Comment:
why does this need to change?

Reply:
a couch doc is accessed during this test.  For some unknown reason it wasn't an issue in python 2.

Reply:
:+1: